Autopilot/Src/MPXV7002DP.cpp

```diff
@@ -0,0 +1,65 @@
+#include "airspeed.hpp"
+#include "adc.h"
+#include <stddef.h>
+#include <cmath>
+
+#define VS 5
+#define DENSITY_AIR 1.204
+#define ADC_RESOLUTION 4096
+
+
+MPXV7002DP::MPXV7002DP() {
+    //constructor
+    MX_ADC2_Init();
```

Comment:
There's more to be done here. Remember from your bootcamp, you needed to put the ADC in constant conversion modeby calling hal_adc_start or something.